build is failing since the update of Dart Sass to 1.9.0

Culprit: https://github.com/bazelbuild/rules_sass/commit/38989d69ef3ba5847640f007fee5cc489be6ede9
CI: https://buildkite.com/bazel/rules-sass
Logs: https://buildkite.com/bazel/rules-sass/builds/92#903efab4-73fb-40d0-8455-18337a6706c3

Sorry about that (I wonder why it's not public).
Copying the logs here:

```
ERROR: /Users/buildkite/builds/buildkite-imacpro-1-1/bazel/rules-sass/examples/hello_world/BUILD:14:1: Couldn't build file examples/hello_world/main-no-sourcemap.css: SassCompiler examples/hello_world/main-no-sourcemap.css failed (Exit 1)
--
  | WARNING: source-map-support module not installed.
  | Stack traces from languages like TypeScript will point to generated .js files.
  |  
  | failed to load main  Error: Cannot find module 'sass/sass.js'
  | looked in:
  | sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/183/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass.runfiles/sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/183/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass.runfiles/io_bazel_rules_sass/node_modules/sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/183/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass.runfiles/build_bazel_rules_sass_compiletime_deps/node_modules/sass/sass.js
  | at Function.module.constructor._resolveFilename (/private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass_bin_loader.js:258:15)
  | at Function.Module._load (module.js:466:25)
  | at Object.<anonymous> (/private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass_bin_loader.js:290:24)
  | at Module._compile (module.js:635:30)
  | at Object.Module._extensions..js (module.js:646:10)
  | at Module.load (module.js:554:32)
  | at tryModuleLoad (module.js:497:12)
  | at Function.Module._load (module.js:489:3)
  | at Function.Module.runMain (module.js:676:10)
  | at startup (bootstrap_node.js:187:16)
  | ERROR: /Users/buildkite/builds/buildkite-imacpro-1-1/bazel/rules-sass/examples/hello_world/BUILD:6:1: Couldn't build file examples/hello_world/main.css: SassCompiler examples/hello_world/main.css failed (Exit 1)
  | WARNING: source-map-support module not installed.
  | Stack traces from languages like TypeScript will point to generated .js files.
  |  
  | failed to load main  Error: Cannot find module 'sass/sass.js'
  | looked in:
  | sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/182/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass.runfiles/sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/182/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass.runfiles/io_bazel_rules_sass/node_modules/sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/182/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass.runfiles/build_bazel_rules_sass_compiletime_deps/node_modules/sass/sass.js
  | at Function.module.constructor._resolveFilename (/private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass_bin_loader.js:258:15)
  | at Function.Module._load (module.js:466:25)
  | at Object.<anonymous> (/private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass_bin_loader.js:290:24)
  | at Module._compile (module.js:635:30)
  | at Object.Module._extensions..js (module.js:646:10)
  | at Module.load (module.js:554:32)
  | at tryModuleLoad (module.js:497:12)
  | at Function.Module._load (module.js:489:3)
  | at Function.Module.runMain (module.js:676:10)
  | at startup (bootstrap_node.js:187:16)
  | ERROR: /Users/buildkite/builds/buildkite-imacpro-1-1/bazel/rules-sass/examples/nested/BUILD:6:1: Couldn't build file examples/nested/dir/main.css: SassCompiler examples/nested/dir/main.css failed (Exit 1)
  | WARNING: source-map-support module not installed.
  | Stack traces from languages like TypeScript will point to generated .js files.
  |  
  | failed to load main  Error: Cannot find module 'sass/sass.js'
  | looked in:
  | sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/180/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass.runfiles/sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/180/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass.runfiles/io_bazel_rules_sass/node_modules/sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/180/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass.runfiles/build_bazel_rules_sass_compiletime_deps/node_modules/sass/sass.js
  | at Function.module.constructor._resolveFilename (/private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass_bin_loader.js:258:15)
  | at Function.Module._load (module.js:466:25)
  | at Object.<anonymous> (/private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/execroot/io_bazel_rules_sass/bazel-out/host/bin/sass/sass_bin_loader.js:290:24)
  | at Module._compile (module.js:635:30)
  | at Object.Module._extensions..js (module.js:646:10)
  | at Module.load (module.js:554:32)
  | at tryModuleLoad (module.js:497:12)
  | at Function.Module._load (module.js:489:3)
  | at Function.Module.runMain (module.js:676:10)
  | at startup (bootstrap_node.js:187:16)
  | ERROR: /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/external/io_bazel_skydoc/skydoc/sass/BUILD:7:1: Couldn't build file external/io_bazel_skydoc/skydoc/sass/main.css: SassCompiler external/io_bazel_skydoc/skydoc/sass/main.css failed (Exit 1)
  | WARNING: source-map-support module not installed.
  | Stack traces from languages like TypeScript will point to generated .js files.
  |  
  | failed to load main  Error: Cannot find module 'sass/sass.js'
  | looked in:
  | sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/181/execroot/io_bazel_rules_sass/bazel-out/host/bin/external/io_bazel_rules_sass/sass/sass.runfiles/sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/181/execroot/io_bazel_rules_sass/bazel-out/host/bin/external/io_bazel_rules_sass/sass/sass.runfiles/io_bazel_rules_sass/node_modules/sass/sass.js,
  | /private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/sandbox/darwin-sandbox/181/execroot/io_bazel_rules_sass/bazel-out/host/bin/external/io_bazel_rules_sass/sass/sass.runfiles/build_bazel_rules_sass_compiletime_deps/node_modules/sass/sass.js
  | at Function.module.constructor._resolveFilename (/private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/execroot/io_bazel_rules_sass/bazel-out/host/bin/external/io_bazel_rules_sass/sass/sass_bin_loader.js:258:15)
  | at Function.Module._load (module.js:466:25)
  | at Object.<anonymous> (/private/var/tmp/_bazel_buildkite/adc6d8f71d76534133fa7ad4d9374f0d/execroot/io_bazel_rules_sass/bazel-out/host/bin/external/io_bazel_rules_sass/sass/sass_bin_loader.js:290:24)
  | at Module._compile (module.js:635:30)
  | at Object.Module._extensions..js (module.js:646:10)
  | at Module.load (module.js:554:32)
  | at tryModuleLoad (module.js:497:12)
  | at Function.Module._load (module.js:489:3)
  | at Function.Module.runMain (module.js:676:10)
  | at startup (bootstrap_node.js:187:16)
  | INFO: From Linking external/protobuf/libprotobuf_lite.a [for host]:
  | /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: bazel-out/host/bin/external/protobuf/libprotobuf_lite.a(arenastring.o) has no symbols
  | /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: bazel-out/host/bin/external/protobuf/libprotobuf_lite.a(atomicops_internals_x86_msvc.o) has no symbols
  | /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib: file: bazel-out/host/bin/external/protobuf/libprotobuf_lite.a(io_win32.o) has no symbols
  | INFO: From ProtoCompile external/protobuf/python/google/protobuf/any_pb2.py [for host]:
  | external/protobuf/python: warning: directory does not exist.
  | INFO: Elapsed time: 13.047s, Critical Path: 3.63s
  | INFO: 184 processes: 181 remote cache hit, 3 darwin-sandbox.
  | FAILED: Build did NOT complete successfully
  | At least one test failed or was flaky.
```

According to Buildkite, it worked before https://github.com/bazelbuild/rules_sass/commit/38989d69ef3ba5847640f007fee5cc489be6ede9

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -3,9 +3,9 @@ workspace(name = "io_bazel_rules_sass")
 # To use the JavaScript version of Sass, we need to first install nodejs
 http_archive(
     name = "build_bazel_rules_nodejs",
-    url = "https://github.com/bazelbuild/rules_nodejs/archive/0.8.0.zip",
-    strip_prefix = "rules_nodejs-0.8.0",
-    sha256 = "4e40dd49ae7668d245c3107645f2a138660fcfd975b9310b91eda13f0c973953",
+    url = "https://github.com/bazelbuild/rules_nodejs/archive/0.10.0.zip",
+    strip_prefix = "rules_nodejs-0.10.0",
+    sha256 = "2f77623311da8b5009b1c7eade12de8e15fa3cd2adf9dfcc9f87cb2082b2211f",
 )
 load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")
 node_repositories(package_json = [])

